### ISSUE TITLE
Ruby 2.4 has gone out of support, revisiting ruby formating under ruby 2.6 

### ISSUE THREAD

**Issue by blijblijblij** [blijblijblij/devenv#15]

**Comment by blijblijblij** [comment]
hmm

```
Step 16/23 : RUN gem install   ruby-beautify2   rubocop   rubocop-performance   rubocop-rails   rubocop-rspec   sass
 ---> Running in 5741ac1d2a21
`/usr/local/bundle/gems/ruby-beautify2-0.98.0/bin/rbeautify` does not exist, maybe `gem pristine ruby-beautify2` will fix it?
```

seems related to https://github.com/jirutka/ruby-beautify2/issues/1

**Comment by blijblijblij** [comment]
Fixed #15 the ugly way, should supply to fix

**Comment by blijblijblij** [comment]
https://github.com/blijblijblij/devenv/commit/2ca05217500de2e801a933464f87fe1ff025ba88#diff-3254677a7917c6c01f55212f86c57fbfR80-R84
